astropy/units/tests/test_units.py

```diff
@@ -148,6 +148,22 @@ def test_represents():
     assert blabla.decompose().bases == [u.s]
 
 
+@pytest.mark.parametrize(
+    "represents, match_",
+    [
+        ("not_a_unit", "did not parse"),
+        ("mmm", "did not parse"),
```

Comment:
What's the difference between these two? Maybe you should use `pytest.param()` with a custom id?

Reply:
I wanted one that looked more like a unit, but you're right, no real difference.